.github/workflows/sync-demo-signer-api.yml

```diff
@@ -0,0 +1,81 @@
+# Sync the demo signer api to a separate repo
+name: Sync demo signer api
+
+permissions:
+  contents: write
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "apps/signer/**"
+
+jobs:
+  sync:
+    env:
+      ENABLED_CHAINS: 31337
+      DEFAULT_CHAIN_ID: 31337
```

Comment:
if we have a default chain id already selected by default (base), do we need to pass it here?

Reply:
removed